README.rst

```diff
@@ -122,9 +122,40 @@ this is the initial login, the code because this profile is configured
 for use with a passcode device such as a YubiKey. Now when we login
 in to test we are prompted for nothing::
 
-    $ aws --profile prod login
+    % aws --profile prod login
 
 Even if the IdP session has expired in this case we will not be
 prompted for a password because it is stored in the Keyring. The
 user will receive either a phone call or a push to the default
 device configured for Duo to permit authenticating.
+
+Known Issues
+------------
+
+**Unable to authenticate after changing password**
+
+A known problem takes place after changing the underlying password.
+(At Illinois, passwords happen to be stored in Active Directory,
+but the problem probably takes place with other password stores.)
+A subsequent login in to the application fails; a workaround is to
+change the **Enable Keyring** setting to False; the user can then
+type in the password at the prompt.
+
+You can change the data stored in your keyring as follows:
+
+    % keyring set awscli_login username@hostname_of_your_IdP
+
+You may also retrieve your password with this command:
+
+    % keyring get awscli_login username@hostname_of_your_IdP
+
+You may be prompted for your user login password by your operating
+system, depending on how you have your key store configured.
+
+**Problems under Windows**
+
+The Auto-renewal of AWS credentials doesn't work under native Windows
```

Comment:
Instead of *"doesn't work"* maybe we could say does not *"currently support the Windows PowerShell or Command Prompt"*